python/ray/experimental/serve/global_state.py

```diff
@@ -105,15 +112,32 @@ def init_or_get_http_server(self,
             self.refresh_actor_handle_cache()
         return self.actor_handle_cache["http_server"]
 
-    def init_or_get_router(self):
-        if "queue_actor" not in self.actor_handle_cache:
+    def _get_queueing_policy(self, default_policy):
```

Comment:
can you add documentation for this function? i'm a bit confused on what this does

Reply:
Is this still needed